operator/v1/types_ingress.go

```diff
@@ -188,7 +188,6 @@ type LoadBalancerStrategy struct {
 	// scope indicates the scope at which the load balancer is exposed.
 	// Possible values are "External" and "Internal".  The default is
 	// "External".
-	// +optional
```

Comment:
Looks reasonable to me, but we need clarification from @deads2k regarding https://github.com/openshift/api/pull/312/files#r294390499.

Reply:
Since the package default is Optional, does this need to be explicitly required?

Reply:
I marked this explicitly required and moved the defaulting docs up to the actual point of defaulting (the optional `loadBalancer` field).